cmd/konnectd/serve.go

```diff
@@ -178,3 +190,24 @@ func serve(cmd *cobra.Command, args []string) error {
 	logger.Infoln("serve started")
 	return srv.Serve(ctx)
 }
+
+func envOrDefault(name string, def string) string {
```

Comment:
Imo if this is added, all commandline args should use it. Maybe the env stuff can be removed from this PR and instead get its own PR for its own discussion (and then also its own commit, since its basically a new feature).

Reply:
This would introduce back more if/else boilerplate, so lets keep it with this PR?